frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -56,6 +59,43 @@ class PopupCreateFile extends React.Component {
     document.removeEventListener(CUSTOM_EVENT.APP_CUSTOM_EVENT_LISTENER, this.customEventReducer)
   }
 
+  componentDidUpdate (prevProps, prevState, snapshot) {
+    const { state, props } = this
+
+    if (state.uploadedFiles.length === state.uploadFiles.length && state.uploadStarted) {
+      let uploadedFiles = state.uploadedFiles
```

Comment:
this assignation is useless since you reassign it 2 lines bellow

Reply:
Also, don't assign a state property to a variable of the same name. It's not very comprehensive and error prone when you have a state and a local var of the same name. If you expect to modify it, it's name should reflect what the modification is about.
Here, I think you could call it uploadedFileFailedList